qdrant/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bee8e975-219c-4673-83c0-9fa68b973298",
+  "app_id": "qdrant",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "A high-performance vector search engine/database.",
+    "title": "Qdrant",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "A Qdrant Integration Dashboard for Datadog",
+        "image_url": "images/qdrant_dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::",
+      "Offering::",
+      "Queried Data Type::",
+      "Submitted Data Type::"
```

Comment:
these are required to be filled out. If you do not pull data out of datadog, please remove "Queried Data Type::",

Reply:
see here for tags https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags